dictionaries/sr_Cyrl/CHANGELOG.md

```diff
@@ -1,17 +1,20 @@
 # Change Log
 
-## 1.0.0 (2023-01-05)
+## 1.0.1 (2032-01-07)
+
+* removed superfluous locales
```

Comment:
This file is auto-generated from the commits.

Reply:
Got it, thanks.